site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
-  the dashboard page only shows groups that you're a direct member of (for example, if you are a
-  member of Group A and Group A is a member of Group B, you are not a direct member of Group B, and
-  Group B is not shown on the drop-down list).
-- Once you finish the group publishing setup, it can take up to 30 minutes for developers in your
-  group to see the changes.
-- Your domain is the default domain of the new group publisher account. A group publisher can
-  publish items either to the public or to its default domain. A publisher cannot publish to other
-  domains.
-- A developer can act on behalf of multiple publishers if the developer is a member of multiple
-  Google Groups linked to publishers.
-- If a merchant account is needed or is already set up, any member of the group can choose to
-  re-link their own merchant accounts to the publisher on the dashboard page.
-- When you want to create new store items, you will have the opportunity to choose whether to make
-  them part of your personal publisher account or the group publisher account before you upload the
-  item.
+* You can only set up group publishing once and create one group publisher account.
+* You must be the owner or manager of a Google Group to link the group. The drop-down list on the dashboard page only shows groups that you're a direct member of (for example, if you are a member of Group A and Group A is a member of Group B, you are not a direct member of Group B, and Group B is not shown on the drop-down list).
+* Once you finish the group publishing setup, it can take up to 30 minutes for developers in your group to see the changes.
+* A developer can act on behalf of multiple publishers if the developer is a member of multiple Google Groups linked to publishers.
+* If a merchant account is needed or is already set up, any member of the group can choose to re-link their own merchant accounts to the publisher on the dashboard page.
+* Before you upload a new Chrome Web Store item, check that you have selected the publisher you want to own that item. This may be your personal publisher account or any group publisher account you belong to.
 
 ## Move existing items to a Group Publisher account
 
-Once you set up a Group Publisher account (or once you're added as a member of a Google Group linked
-to a Group Publisher account), you can transfer your own items to the Group Publisher.
+Once you set up a group publisher (or once you're added as a member of a Google Group linked to a group publisher), you can transfer your own items to the group publisher.
 
-Remember that once you move items to a Group Publisher account, you won't be able to move them back
-to your personal publishing account.
+<p class="note">Once you move items to a Group Publisher account, you can't move them back to your personal publishing account.</p>
 
-To transfer items to a Group Publisher account:
+To transfer items to a group publisher account you need to use the old developer dashboard:
```

Comment:
no, Snapcat supports it now. Plus the old developer is gone for good. Please don't mention it anymore.

Reply:
Done